src/Copilot.test.ts

```diff
@@ -1,9 +1,14 @@
-import { Copilot } from '@/Copilot';
-import { StepPerformer } from '@/actions/StepPerformer';
-import { CopilotError } from '@/errors/CopilotError';
-import { Config } from "@/types";
-import fs from "fs";
-import { mockCache, mockedCacheFile } from "./test-utils/cache";
+import {Copilot} from '@/Copilot';
+import {StepPerformer} from '@/actions/StepPerformer';
+import {CopilotError} from '@/errors/CopilotError';
+import {Config} from "@/types";
+import {mockCache, mockedCacheFile} from "./test-utils/cache";
+import {
+    customActionsCategory,
+    actionsCategory2,
+    actionsCategory,
```

Comment:
naming is confusing (both refer to actions)
use `barCategory` and `bazCategory1`, `bazCategory2` or something? just not to mention actions in both categories